[2020-08-26 11:51:48,321] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: submit_data.spark_submit2 2020-08-24T00:03:00+00:00 [queued]>
[2020-08-26 11:51:48,332] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: submit_data.spark_submit2 2020-08-24T00:03:00+00:00 [queued]>
[2020-08-26 11:51:48,332] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-08-26 11:51:48,332] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-08-26 11:51:48,332] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-08-26 11:51:48,554] {taskinstance.py:900} INFO - Executing <Task(SparkSubmitOperator): spark_submit2> on 2020-08-24T00:03:00+00:00
[2020-08-26 11:51:48,561] {standard_task_runner.py:53} INFO - Started process 76685 to run task
[2020-08-26 11:51:48,759] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: submit_data.spark_submit2 2020-08-24T00:03:00+00:00 [running]> bhakti-Vostro-3558
[2020-08-26 11:51:48,781] {logging_mixin.py:112} INFO - [2020-08-26 11:51:48,781] {base_hook.py:87} INFO - Using connection to: id: spark_local. Host: local, Port: None, Schema: None, Login: None, Password: None, extra: None
[2020-08-26 11:51:48,782] {logging_mixin.py:112} INFO - [2020-08-26 11:51:48,782] {spark_submit_hook.py:325} INFO - Spark-Submit cmd: spark-submit --master local --jars /home/bhakti/airflow/jars/mysql-connector-java-5.1.47-bin.jar --name airflow-spark /home/bhakti/airflow/spark_jobs/write_mysql.py
[2020-08-26 11:51:48,796] {logging_mixin.py:112} INFO - [2020-08-26 11:51:48,796] {spark_submit_hook.py:479} INFO - /usr/local/bin/find-spark-home: line 40: python: command not found
[2020-08-26 11:51:48,797] {logging_mixin.py:112} INFO - [2020-08-26 11:51:48,797] {spark_submit_hook.py:479} INFO - /usr/local/bin/spark-submit: line 27: /bin/spark-class: No such file or directory
[2020-08-26 11:51:48,807] {taskinstance.py:1145} ERROR - Cannot execute: spark-submit --master local --jars /home/bhakti/airflow/jars/mysql-connector-java-5.1.47-bin.jar --name airflow-spark /home/bhakti/airflow/spark_jobs/write_mysql.py. Error code is: 127.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/airflow/models/taskinstance.py", line 983, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.8/dist-packages/airflow/contrib/operators/spark_submit_operator.py", line 187, in execute
    self._hook.submit(self._application)
  File "/usr/local/lib/python3.8/dist-packages/airflow/contrib/hooks/spark_submit_hook.py", line 403, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local --jars /home/bhakti/airflow/jars/mysql-connector-java-5.1.47-bin.jar --name airflow-spark /home/bhakti/airflow/spark_jobs/write_mysql.py. Error code is: 127.
[2020-08-26 11:51:48,810] {taskinstance.py:1189} INFO - Marking task as FAILED.dag_id=submit_data, task_id=spark_submit2, execution_date=20200824T000300, start_date=20200826T062148, end_date=20200826T062148
[2020-08-26 11:51:58,102] {logging_mixin.py:112} INFO - [2020-08-26 11:51:58,102] {local_task_job.py:103} INFO - Task exited with return code 1
